src/shared/ContentsTable/TableEntries/PullEntries/PullFileEntry.jsx

```diff
@@ -13,6 +15,9 @@ function PullFileEntry({
   urlPath,
   displayType,
 }) {
+  const { search } = useLocation()
+  const queryParams = qs.parse(search, { ignoreQueryPrefix: true })
+
```

Comment:
Same thing here, you can pass a filters prop and have them passed down from the contents table hook that is using these components

Reply:
There is no filters in the PullFileEntry, I'm unclear on how I'd be able to extend this like the one above. Could you clarify?